contents/blog/writing-for-developers.md

```diff
@@ -0,0 +1,111 @@
+---
+date: 2021-05-20
+title: Writing for Developers
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [James Hawkins](https://twitter.com/james406), Co-Founder/CEO of PostHog
+
+As a technical founder, one of the most valuable skills is hitting the front page of [HackerNews](https://news.ycombinator.com).
+
+Out of five posts that I hoped would get there, four did. Look mom, lots of traffic:
+
+![Blog traffic for PostHog](../images/blog/writing-for-developers/posthog-blog-traffic.jpg)
+
+I don't claim to be a well-established writer. Amongst other things, I'm inconsistent with publishing and being British means I wrestle with US English. On the flipside, I've spent an embarassingly large amount of time reading content that engineers find engaging.
+
+This is what I have learned thus far. I've created a framework for those also working at developer-focused companies, writing for the same audience as me.
+
+Disclaimer: my company doesn't care about appealing (directly) to CTOs. If you work at Top Down Enterprise Technology Corp, just do the opposite of what I say.
+
+## Be human
+
+Personal theory: humans evolved to care about other humans - it's how we survived as a race through prehistoric times. Companies are mostly a bunch of humans, so if you get that across then people will want to read your writing.
+
+*Do:*
+
+* Give character to the _humans_ involved in what you write about. Go into unexpected detail. Tell [stories](https://hbr.org/2014/07/how-to-tell-a-great-story). They don't need to be impressive - the goal is to make things real.
+* Make it real by adding real life photos, chat logs, social media posts, social media profiles, etc. 
+* Add humor - be self deprecating where appropriate. No one is perfect. We make mistakes, and being willing to open up builds trust. 
+
+*Don't:*
+
+* Give clichéd startup advice. If what you're writing is generic advice that 90% of people already know, and is something that PostHog just followed, then assume the audience will already know it. 
+
+## Be concise
+
+Think about your audience here, folks.
+
+The life of a developer involves rapidly finding answers to sometimes-what-feels-like-endless problems. Whilst not universal, my grand theory here is that many have an impatient reading style developed by scanning for relevant information all day... probably something to do with googling Stack Overflow a lot.
+
+<iframe src="https://giphy.com/embed/10eJOwQ9BKrF72" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+
+"I would have written a shorter letter, but I did not have the time." - [Blaise Pascal](https://en.wikipedia.org/wiki/Blaise_Pascal)
+
+Most of your time should be spent *re*writing.
+
+*Do:*
+
+* Get to the point as quickly as possible. 
```

Comment:
Strunk and White's Element of Style sums this up nicely: "Omit needless words"

Reply:
Ooh, done!